src/components/TodosList/TodosList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import '../../styles/animation.scss';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[];
+  clearIsPressed: boolean;
```

Comment:
Consider renaming 'clearIsPressed' to a more descriptive name like 'isClearPressed' for better readability 📖